docs/querying/sql-functions.md

```diff
@@ -1103,11 +1124,32 @@ Returns the following:
 
 ## EXTRACT
 
-`EXTRACT(<TIME_UNIT> FROM <TIMESTAMP>)`
+Extracts the value of some unit from the timestamp.
 
-**Function type:** [Scalar, date and time](sql-scalar.md#date-and-time-functions)
+* **Syntax:** `EXTRACT(unit FROM timestamp_expr)`
+* **Function type:** Scalar, date and time
+
+<details><summary>Example</summary>
+
+The following example extracts the year from the `__time` column from the `taxi-trips` datasource.
+
+```sql
+SELECT 
+  "__time" AS "original_time",
+  EXTRACT(YEAR FROM "__time" ) AS "year"
+FROM "taxi-trips"
+LIMIT 1
+```
+
+Returns the following:
+
+| `original_time` | `year` |
+| -- | -- |
+| `2013-08-01T08:14:37.000Z` | `2013` |
```

Comment:
```suggestion
| `2013-08-01T08:14:37.000Z` | `2,013` |
```
The query as written returns 2,103.  Another option would be to cast the year as string:
`CAST(EXTRACT(YEAR FROM "__time" ) AS varchar) AS "year"`

It's weird this may just be the display showing 2,013 if you show the full value, it doesn't use the commas.

Reply:
Will ignore since in context of year `2013` (inspect value) is more relevant than initial UI